rfcs/0164-bootspecv2.md

```diff
@@ -0,0 +1,300 @@
+---
+feature: bootspec_v2
+start-date: 2022-11-01
+author: Ryan Lahfa
+co-authors:
+shepherd-team: 
+shepherd-leader:
+---
+
+# Summary
+[summary]: #summary
+
+Bootspec v2 is the second revision of the Bootspec document, introduced in RFC-0125.
+
+These facts are used as the primary input for bootloader backends like systemd-boot and grub, for creating files in `/boot/loader/entries/` and `grub.cfg`.
+
+In this proposal, we aim to tackle known weaknesses of Bootspec v1, namely:
```

Comment:
Nit: the list order should reflect the order of the sections below?

Reply:
Done.